reports/performance/_model.qmd

```diff
@@ -242,6 +242,222 @@ for (township in townships_to_iterate) {
 ```
 :::
 
+## How do Sales Compare to Estimates?
+
+```{r _model_sales_vs_estimates_township, fig.width=8, fig.height=14}
+sales_vs_estimates <- model_performance_test %>%
+  filter(
+    triad_code == run_triad_code,
+    geography_type == "township_code",
+    by_class == FALSE
+  ) %>%
+  select(
+    geography_type,
+    geography_id,
+    sale_fmv_q25,
+    sale_fmv_median,
+    sale_fmv_q75,
+    estimate_fmv_q25,
+    estimate_fmv_median,
+    estimate_fmv_q75
+  )
+
+reshaped_data <- sales_vs_estimates %>%
+  mutate(township = ccao::town_convert(geography_id)) %>%
+  pivot_longer(
+    cols = -c(geography_type, geography_id, township),
+    names_to = "type",
+    values_to = "value"
+  ) %>%
+  mutate(
+    category = ifelse(
+      grepl("sale", type),
+      paste(township, " - Sale"),
+      paste(township, " - Estimate")
+    )
+  ) %>%
+  select(category, value) %>%
+  mutate(
+    space_maker = as.numeric(factor(category)),
+    space_maker = space_maker + (space_maker - 1) %/% 2
+  )
+
+line_ends <- reshaped_data %>%
+  group_by(space_maker, category) %>%
+  summarize(Start = min(value), End = max(value)) %>%
+  mutate(
+    color_class = ifelse(grepl("Sale", category), "Sale", "Estimate"),
+    # Set for legend re-ordering
+    color_class = factor(color_class, levels = c("Sale", "Estimate"))
+  )
+
+ggplot() +
```

Comment:
**issue:** You can make this plot far easier using something like `facet_wrap()` or `facet_grid()`. No need for spacers. Also, move the legend to the bottom of the plot. You could also add information here using a box or violin plot, though @ccao-jardine may disagree.

Reply:
Might also be worth playing with the "position" option in ggplot to condense the bars within townships.

Reply:
Added graphs using `facet_grid()`.